### DISCUSSION TITLE
Previous Notes

### DISCUSSION THREAD

**Discussion by encomun** [emacs-citar/citar#723] [Q&A] [answered]
Is there a way for citar to link already existing notes to keys? I've been able to successfully create new notes, but I'm not sure how to bring in previously written notes. I tried naming the file as [key].org and putting it in my notes directory, but citar doesn't seem to pick it up.

**Comment by bdarcus** [comment, marked as answer]
> I tried naming the file as [key].org and putting it in my notes directory, but citar doesn't seem to pick it up.

You're using the default file-based notes source (not `citar-org-roam`, `citar-denote`, etc.)?

If yes, it _should_ work.

**Reply by encomun** [reply]
Yep, after restarting a couple times, it worked.

I am interested in citar-denote. Would this allow files to have titles beyond just key.org (or key.md)?

**Reply by pprevos** [reply]
Hi, with [Citar-Denote](https://github.com/pprevos/citar-denote) you can link notes created with the [Denote package](https://protesilaos.com/emacs/denote) to a bibliography item. It does this by reading all files marked as bibliography files and reading the reference field inside the document. It can manage Org Mode, markdown and plain text.

In Denote, filenames have a naming convention, which is `YYYYMMDDTHHMMSS--note-title__keyword1_keyword2.ext`. The note title or keywords can be anything you like, the first 15 are the unique identifier. Hope that helps.

**Comment by bdarcus** [comment]
I don't actually know how it works.

@pprevos?
